utils/netutils/netutils.go

```diff
@@ -213,6 +213,23 @@ func ParseTagRanges(ranges string, tagType string) ([]TagRange, error) {
 	return tagRanges, nil
 }
 
+// IsTagInRange verifies if the passed pktTag adheres to the valueRange
+func IsTagInRange(pktTag int, valueRangeStr string, encap string) bool {
+	// Parse the valueRange to form a []TagRange object
+	valueRanges, err := ParseTagRanges(valueRangeStr, "vlan")
+	if err != nil {
+		log.Errorf("Not able to pass valueRange. Err: %s", err)
+		return false
+	}
+
+	for _, valueRange := range valueRanges {
```

Comment:
@vijayvikrant - as we talked offline, let's see if the call to ParseRange can ba avoided in IsTagInRange() i.e. pass the range from pre-parsed (tenant) to avoid doing this twice